x/pylons/types/message_google_iap_get_coins.go

```diff
@@ -61,6 +61,7 @@ func (msg *MsgGoogleInAppPurchaseGetCoins) ValidateBasic() error {
 	if err != nil {
 		return err
 	}
+
 	if msg.PurchaseToken != jsonData["purchaseToken"] {
 		return fmt.Errorf("purchaseToken does not match with receipt data")
```

Comment:
Can we consider using(adding) a blockchain specific error for this instead of using fmt.Errorf